content/docs/other/a_word_on_backward_compatibility.md

```diff
@@ -30,9 +33,10 @@ This is the current list of feature flags:
 | 0.53                                   | 0.53      | Will activate register\_cmdb\_top\_scope. And all things 0.51 and down activated. |
 | register\_cmdb\_top\_scope             | 0.53      | Will register all cmdb top scope variables automatically in the templates. |
 | 0.51                                   | 0.51      | Will load Rex::Constants and the CMDB by default. And all things 0.47 and down activated. |
-| no\_autodie                            | 1.0       | Will disable autodie feature. |
-| no\_path\_cleanup                      | 0.44      | Rex cleans the path before executing a command. With this feature Rex doesn't cleanup the path. |
+| disable\_taskname\_warning             | 0.47      | |
 | verbose\_run                           | 0.47      | Explicitly output "Successfully executed" or "Error executing" messages for run() commands. |
+| no\_cache                              | 0.46      | |
```

Comment:
Proposed description:

Disable caching

Reply:
Hmm. That's still rather terse: it immediately triggers the question "which caches"? What I'd like is at least a list of Rex::Commands that are influenced by this setting; at the bare minimum a list of examples.

Reply:
I'm afraid there's no exhausting documentation yet about what exactly gets cached during an execution, so we cannot simply just point somewhere (this would be ideal though).

Mostly remote discovery is cached (hardware, OS, shell, etc.), so rex don't need to redo that for each step again from scratch. Basically everything that can be used in a template as a built-in variable, is cached. If a cache file is configured, then those even get preserved between rex runs.

So the bar minimum for a list of examples might be:

Disable caching (like discovery results of remote OS, hardware, shell, etc.)

Reply:
Added. Thanks!